includes/3rd-party/wordpress-importer.php

```diff
@@ -5,6 +5,26 @@
  * @package 3rd-Party
  */
 
+/**
+ * Track when dummy data is imported but don't track any events that could fire during import.
+ *
+ * @return bool
+ */
+function sensei_wordpress_importer_usage_tracking_dummy_data() {
+	// Log the import event if we're importing the dummy data.
+	$global_importer = isset( $GLOBALS['wp_import'] ) ? $GLOBALS['wp_import'] : false;
+	if (
+		$global_importer
+		&& 'WP_Import' === get_class( $global_importer )
+		&& 'http://demo.sensei.com/' === $global_importer->base_url
+	) {
+		sensei_log_event( 'data_import', [ 'dummy_data' => 1 ] );
+
+		// Disable tracking of other events.
+		add_filter( 'sensei_log_event', 'sensei_wordpress_importer_disable_event_tracking' );
```

Comment:
I'm using a helper function here instead of `__return_false` so that it doesn't interfere with other attempts at disabling usage tracking when we go to `remove_filter()` below.